public/app/features/alerting/unified/utils/access-control.ts

```diff
@@ -47,6 +47,11 @@ export const notificationsPermissions = {
   },
 };
 
+export const provisioningPermissions = {
```

Comment:
nit: Is it used anywhere?

Reply:
ops! I forgot using it 😄, I want to use it in the `getRulesAccess` method in `access-control.ts` to be consistent with the rest of rules RBAC checks.
I'm going to push this change.Thank you! @konrad147